mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaError.java

```diff
@@ -1,10 +0,0 @@
-package io.mosip.esignet.plugin.mock.dto;
-
-import lombok.Data;
-
-@Data
-public class ReCaptchaError {
```

Comment:
Why this and reponse is removed ?

Reply:
Because we have started to use mosip captcha validation service.